pootle/core/initdb.py

```diff
@@ -23,11 +24,19 @@
 
 
 logger = logging.getLogger(__name__)
+re_plurals = re.compile(r'nplurals=(\d); plural=(.+)$')
+
+
+CLDR_FALLBACK_ALIASES = {
+    # new name: list of fallback names
+    'zh_Hans': ['zh_CN'],
+    'zh_Hant': ['zh_TW', 'zh_HK'],
+}
 
 
 class InitDB(object):
 
-    def init_db(self, create_projects=True):
+    def init_db(self, create_projects=True, cldr=False, aliases=CLDR_FALLBACK_ALIASES):
```

Comment:
Probably it is not necessary to pass `CLDR_FALLBACK_ALIASES` as it is available in the module.